src/components/CowCard/Subheader/Subheader.test.js

```diff
@@ -1,28 +1,25 @@
 import React from 'react'
-import { render } from '@testing-library/react'
+import { render, screen } from '@testing-library/react'
 
-import { generateCow } from '../../../utils'
-import { cowColors } from '../../../enums'
+import { getCowStub } from '../../../test-utils/stubs/cowStub'
+import { moneyString } from '../../../utils/moneyString'
 
 import Subheader from './Subheader'
 
+const cowValueStub = 1000
+
 describe('Subheader', () => {
   let baseProps
 
   beforeEach(() => {
     jest.spyOn(Math, 'random').mockReturnValue(0)
     baseProps = {
       canCowBeTradedFor: false,
-      cow: generateCow({
-        color: cowColors.WHITE,
-        happiness: 0,
-        name: '',
-        baseWeight: 100,
-      }),
+      cow: getCowStub(),
       cowBreedingPen: { cowId1: null, cowId2: null, daysUntilBirth: -1 },
       cowIdOfferedForTrade: '',
       cowInventory: [],
-      cowValue: 1000,
+      cowValue: cowValueStub,
```

Comment:
personal preference/nitpicky semantics (aka feel free to ignore), but i wouldn't consider a primitive value a "stub". i do like having a variable for things like this though. i think it's still simple enough to follow if it's just `cowValue`, or this is when i often use an uppercase const like `COW_VALUE`

Reply:
I agree that using "stub" semantics feels a little odd for primitive values. My concern with `cowValue` is that it's not immediately obvious that it's a test value to assert against (to my eyes), and `CAMEL_CASE_CONSTANTS` are generally used for global application constants (in this project). In this case a variable called `TEST_COW_VALUE` that's local to this file feels the least-weird to me. What do you think?

Reply:
i use uppercase constants for this sort of thing all the time. i don't think you need to add "TEST" to it. when i see an uppercase constant i just know it's more global than the scope of any method, and it should be defined towards the top (which sometimes leads to another file due to import). contextually speaking we know we're in a test file here, and a pretty short one at that. i don't think anyone will be confused about it

Reply:
That seems reasonable to me. https://github.com/jeremyckahn/farmhand/commit/528d8fd04425b78f82ec063967bc74d8907c81d9 switches this to `COW_VALUE`!